data/mongocli-publishsed-branches.yaml

```diff
@@ -0,0 +1,18 @@
+version:
+  published:
+    - 'v0.0.4'
+    - 'v0.0.3'
+  active:
+    - 'v0.0.4'
+    - 'v0.0.3'
+  stable: 'v0.0.3'
+  upcoming: 'v0.0.4'
+git:
+  branches:
+    manual: 'v0.0.4'
+    published:
```

Comment:
I don't know this part of the yaml that well, but does `git/branches/published` need to contain `v0.0.4`?

Reply:
No, because `0.0.4` is an abstraction for display only.

Reply:
Right, in this case, `master` is `0.0.4`